webpack.config.js

```diff
@@ -59,46 +70,22 @@ module.exports = {
             {
                 test: /\.scss$/,
                 use: [
-                    {
-                        loader: MiniCssExtractPlugin.loader,
-                        options: {
-                            publicPath: "../",
-                        },
-                    },
-
                     {
                         loader: "css-loader",
                         options: { sourceMap: devMode, url: true },
                     },
-                    {
-                        loader: 'resolve-url-loader'
-                    },
                     {
                         loader: "sass-loader",
                         options: { sourceMap: true },
                     },
                 ],
             },
-            {
-                test: /\.pug$/,
-                use: {
-                    loader: "@webdiscus/pug-loader",
-                },
-            },
             {
                 test: /\.(jp?g|png|gif|svg)$/i,
-                // type: 'asset/resource',
-                // generator: {
-                //     filename: 'images/[name][ext]'
-                // },
-                use: [
-                    {
-                        loader: "file-loader",
```

Comment:
In Webpack 5 the file-loader is deprecated. Instead of the `file-loader` must be used `asset/resource`.